### DISCUSSION TITLE
AB2E429F-2EAC-1AD3-BF15-2F65F114D116

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#53] [General]
# M 101 (Pinwheel Galaxy)

The Pinwheel Galaxy (also known as Messier 101 or NGC 5457) is an Sc-type spiral galaxy about 24.58 million light-years away in the constellation Ursa Major. It is measured at 170,000 light-years in diameter, and containing an estimated a trillion stars, making it larger than the Milky Way in comparison

https://celestia.mobi/addon?item=AB2E429F-2EAC-1AD3-BF15-2F65F114D116&lang=en

**Comment by exocritic5457** [comment]
I think I speak for myself here.

**Reply by kmjordan122313-lgtm** [reply]
i love it! ^_^

**Reply by kmjordan122313-lgtm** [reply]
this is one of my favorite galaxies.

**Comment by kmjordan122313-lgtm** [comment]
my favorite galaxies.
pinwheel, milky way, and andromeda.
